src/data/prod/lex.lean

```diff
@@ -8,13 +8,7 @@ import order.synonym
 /-!
 # Lexicographic order
 
-This file defines the lexicographic relation for pairs and dependent pairs of orders, partial orders
-and linear orders.
-
-## Main declarations
-
-* `lex α`: A type synonym of `α` to equip it with its lexicographic order.
-* `prod.lex.<pre/partial_/linear_>order`: Instances lifting the orders on `α` and `β` to `α ×ₗ β`.
```

Comment:
These instances are still in this file, right? Did we decide to remove instances from the module docs?

Reply:
It's just that the file very small now, so it doesn't really make sense to have a Main Declarations section which just lists all declarations.

Reply:
Thanks for restoring the section! My thinking on this is that even for tiny files, it's still easier (esp. for newcomers) to browse the module docstring on the online docs than to dig through all the types written in Lean.

bors r+